lib/cyclone-core/src/request.rs

```diff
@@ -38,6 +38,7 @@ pub trait CycloneRequestable {
     type Response;
 
     fn execution_id(&self) -> &str;
+    fn kind(&self) -> &str;
```

Comment:
I'm a bit surprised we never had this, but I think it's the way we transmitted the requests that allowed us to know the kind of function (that it, the NATS subject that we subscribe to let us know). This does feel clearer and understand that it becomes a lot more required *inside* the running `lang-js` now.